docs/reference/components/sources/http.cue

```diff
@@ -107,8 +107,38 @@ components: sources: http: {
 				}
 			}
 		}
+		url_path: {
```

Comment:
What would you think of just calling this `path`? I think, in the context it appears, it'll be pretty clear it refers to an HTTP URL path.